src/adapters/beachfront.js

```diff
@@ -0,0 +1,120 @@
+import Adapter from 'src/adapters/adapter';
+import bidfactory from 'src/bidfactory';
+import bidmanager from 'src/bidmanager';
+import * as utils from 'src/utils';
+import { ajax } from 'src/ajax';
+import { STATUS } from 'src/constants';
+
+const ENDPOINT = '//reachms.bfmio.com/bid.json?exchange_id=';
+
+function BeachfrontAdapter() {
+  var baseAdapter = Adapter.createNew('beachfront'),
+    bidRequest;
+
+  baseAdapter.callBids = function (bidRequests) {
+    if (!bidRequests || !bidRequests.bids || bidRequests.bids.length === 0) {
+      return;
```

Comment:
Any place that is returning in an "error" case should probably log an error as well as register a `NO_BID` response (so that the auction isn't held up waiting).